STYLEGUIDE.md

```diff
@@ -0,0 +1,101 @@
+```
+.. Note::
+
+   This is a note.
+```
+
+## Images/Figures
+
+- Organize images into directories following the Google drive
+- Lower case file names and extensions, separate with hyphens, 3-character extensions
+- For ease of visibility, explanatory images should be full width; expository or those with very simple subjects can be half-width, right aligned
+- Use PNG for all images, except photographs over 100-200 KB as a PNG (use moderately compressed JPEG); avoid sizes above 2000 x 2000 or below 1000 x 1000 if full width, or half of each if half width
+- Always include alt text describing the content of the image
+- No space before, one space ("|") after if full width
+
+
+## Writing (All Languages)
+
+- Follow the [OpenStack General Writing Guidelines](https://docs.openstack.org/doc-contrib-guide/writing-style/general-writing-guidelines.html)
+- Maintain an explanatory rather than expository tone if appropriate for the language (i.e. in English, "How" rather than "What"; e.g. "Spyder can do XXX." -> "To do XXX, you can...")
+- Focus on the user; this is documentation, not promotional material
+- Try to use simple language if possible and avoid buzzwords, jargon and acronyms
+- Make sure capitalization, spacing etc. of proper names and acronyms are correct and consistent per the original (or the canonical localized form is availible)
+- In text, use the full name for file types ("TIFF" and "JPEG", not "TIF" and "JPG"), and always use ``/`` as the path delimiter
+- Must use ISO 8601 (YYYY-MM-DD HH:MM:SS, 24-hour format, with leading zeros) for all numeric dates and times. No convention is imposed for textual forms (e.g. January 1, 2018; 1 de abril de 2019; etc) so long as all three components are provided, the month names are written out in full, and the year is written as 4 digits.
+-  ``.`` must be used as the decimal separator in languages that at least partially use it (i.e. a native speaker would not be unfamiliar with its use), and suggested otherwise at the option of the translator to be consistent with its use in Python and  internet standards, and avoid the textual ambiguity of `,`; regardless of which is chosen, it is more important to use it consistently throughout the docs of a given language
+- If a separator is used between groups of digits for readability, it must be the space to avoid ambiguity per ISO standard conventions
+- Should use only SI/metric for units
+
+
+## Writing (English)
+
+- Standard written English; US spellings are nominally the default for consistency but not required, and any given page should keep to only one convention. Avoid dialect-specific words.
+- AP style unless noted elsewhere here
```

Comment:
What does this mean?

Reply:
Use AP style, unless stated otherwise by this style guide